lib/fizzy/parser_expr.cpp

```diff
@@ -178,6 +178,7 @@ parser_result<Code> parse_expr(const uint8_t* pos)
         case Instr::br:
         case Instr::br_if:
         case Instr::call:
+        case Instr::call_indirect:
```

Comment:
I believe this is wrong, it should also have:
```suggestion
        case Instr::call_indirect:
        {
            uint32_t imm;
            std::tie(imm, pos) = leb128u_decode<uint32_t>(pos);
            push(code.immediates, imm);

                        const uint8_t tableidx{*pos++};
            if (tableidx != 0)
                throw parser_error{"invalid tableidx encountered with call_indirect"};
            break;
 }
```